web/src/message_list_hover.ts

```diff
@@ -13,6 +14,8 @@ export function message_unhover(): void {
         return;
     }
     $current_message_hover.removeClass("can-edit-content can-move-message");
+    clearTimeout(Number($current_message_hover.attr("data-edit-timeout-id")));
+    $current_message_hover.removeAttr("data-edit-timeout-id");
```

Comment:
Let's move `$current_message_hover.removeClass("can-edit-content can-move-message");` after `removeAttr` to keep the DOM operations together.

Reply:
Should we just add a variable to track the current hover timeout id like we do for `$current_message_hover`?